### ISSUE TITLE
1.19 41.0.63 Support

### ISSUE THREAD

**Issue by Roalith** [UnlikePaladin/paladin-furniture-mod#5]
Good day, my child plays using FTB One and we tried every 1.19 Forge version of this mod and it crashes each time. FTB One is stuck using FML 41.0.63 until an unknown date in the future. Could we trouble you for a version that supports this Forge version for 1.19? We have had to resort to other compatible furniture mods in the interim but would love to enjoy this one!

**Comment by UnlikePaladin** [comment]
Hi, the mod is actually compiled for 1.19 and it should work on all available 1.19 versions

**Comment by UnlikePaladin** [comment]
Let me try and run it on that specific forge version, it's quite old however so it might crash

**Comment by UnlikePaladin** [comment]
I've tried it, I can't make it run on that version of Forge because the Cloth Config Version I compiled against requires a newer version of Forge and even if I were to remove that lock it wouldn't work anyway because the Architectury needs a newer version of Forge and with the FTB App you can't change that. I'll make a build for you to use and I'll put it here.

**Comment by Roalith** [comment]
Right, thanks! We have Cloth Config and Architectury in the modpack from FTB (versions for 41.0.63 of course). That's very kind of you to offer, it sounds like some work. Do you have a Patreon or similar set up? I was hoping it was just a minor change but it sounds like this will take a bit more effort than that.

**Comment by UnlikePaladin** [comment]
Don't worry, I'm done already, it wasn't that much work. I don't have a Patreon yet, but when I do the link will be on the Mod's Page, but it will probably be Kofi which is pretty similar except it allows for one time donations. I saw your suggestions on the CF Page, I'm working on those for the next version but meanwhile here's a version of the mod compatible with 41.0.63, I've tested it with the modpack itself to check that it works and it seems to work alright, have fun with your kiddo, happy decorating! (Once you unzip it you'll get the mod file, Github doesn't allow for Jars to be uploaded directly)
[PaladinsFurniture-1.19-UnzipMe.jar.zip](https://github.com/UnlikePaladin/paladins-furniture/files/9650407/PaladinsFurniture-1.19-UnzipMe.jar.zip)
